lib/rubocop/cop/sorbet/mixin/target_sorbet_version.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Sorbet
+      module TargetSorbetVersion
+        def self.included(target)
+          target.extend(ClassMethods)
+        end
+
+        module ClassMethods
+          # The version of the Sorbet static type checker required by this cop
+          def minimum_target_sorbet_static_version(version)
+            @minimum_target_sorbet_static_version = Gem::Version.new(version)
+          end
+
+          def support_target_sorbet_static_version?(version)
+            @minimum_target_sorbet_static_version <= Gem::Version.new(version)
+          end
+        end
+
+        def enabled_for_sorbet_static_version?
+          sorbet_static_version = target_sorbet_static_version_from_bundler_lock_file
+          return false unless sorbet_static_version
+
+          self.class.support_target_sorbet_static_version?(sorbet_static_version)
+        end
+
+        def target_sorbet_static_version_from_bundler_lock_file
+          @target_sorbet_static_version_from_bundler_lock_file ||= read_sorbet_static_version_from_bundler_lock_file
+        end
+
+        # Adapted from https://github.com/rubocop/rubocop/blob/1181d4ebad5f71c586f9514d9c341cdfffc1957d/lib/rubocop/config.rb#L293-L308
+        def read_sorbet_static_version_from_bundler_lock_file
+          lock_file_path = config.bundler_lock_file_path
+
+          return nil unless lock_file_path
+
+          File.foreach(lock_file_path) do |line|
+            # If Sorbet (or one of its frameworks) is in Gemfile.lock or gems.lock, there should be
```

Comment:
This comes from https://github.com/rubocop/rubocop/blob/3bf6d4132736ac228efdaf0e85901acbaa968ce0/lib/rubocop/config.rb#L293-L308

I'd like to keep it the same until hopefully one day it can be deleted (replaced with some Rubocop API that let's you specify min version reqs).